samples/glean/metrics.yaml

```diff
@@ -2,7 +2,7 @@
 # automatically converted to Kotlin code at build time using the `glean_parser`
 # PyPI package.
 
-$schema: http://mozilla.org/schemas/glean/metrics
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
```

Comment:
That's changing due to the version bump for the parsers, right?

Reply:
Yes.